pkgs/development/python-modules/pyscf/default.nix

```diff
@@ -0,0 +1,43 @@
+{ buildPythonPackage, lib, fetchFromGitHub, libcint, libxc, xcfun, blas,
+  numpy, scipy, h5py, python3
+} :
+
+buildPythonPackage rec {
+  pname = "pyscf";
+  version = "1.7.6.post1";
+
+  src = fetchFromGitHub {
+    owner = "pyscf";
+    repo = pname;
+    rev = "f6c9c6654dd9609c5e467a1edd5c2c076f793acc";
+    sha256  = "0xbwkjxxysfpqz72qn6n4a0zr2h6sprbcal8j7kzymh7swjy117w";
+  };
+
+  buildInputs = [
+    libcint
+    libxc
+    xcfun
+    blas
+  ];
+
+  propagatedBuildInputs = [
+    numpy
+    scipy
+    h5py
+    python3
+  ];
+
+  PYSCF_INC_DIR="${libcint}:${libxc}:${xcfun}";
+
+  # The test suite is very sensitive to the file system and built only correctly on tmpfs.
+  # Therefore disabled.
+  doCheck = false;
```

Comment:
Can ofBorg build it when you turn on the tests?

Reply:
I will just push a commit to check this.

Reply:
Seems to work. I suggest keeping it turned on and remove the comment. We never double checked the tmpfs hypothesis.

Reply:
OK, turned them on and removed comment!